django_rq/utils.py

```diff
@@ -10,9 +10,32 @@
 from rq.worker import Worker
 from rq.worker_registration import clean_worker_registry
 
-from .queues import get_connection, get_queue_by_index
+
+from .queues import get_connection, get_queue_by_index, get_scheduler
 from .settings import QUEUES_LIST
 from .templatetags.django_rq import to_localtime
+from django.core.exceptions import ImproperlyConfigured
+
+def get_scheduler_pid(queue):
+    '''Checks whether there's a scheduler-lock on a particular queue, and returns the PID.
+        It Only works with RQ's Built-in RQScheduler.
+        When RQ-Scheduler is available returns False 
+        If not, it checks the RQ's RQScheduler for a scheduler lock in the desired queue
+        Note: result might have some delay (1-15 minutes) but it helps visualizing whether the setup is working correcly
+    '''
+    try:
+        # first try get the rq-scheduler
+        scheduler = get_scheduler(queue.name)  # should fail if rq_scheduler not present
+        return False  # Not possible to give useful information without creating a performance issue (redis.keys())
```

Comment:
This will cause `get_scheduler_pid` to always return `False` if `rq_scheduler` is installed, even though the built in scheduler is used.

I think we should just skip the check for external scheduler altogether.

Reply:
Well, ok. In my tests, running both --with-scheduler and rq-scheduler doesn't end up well.
By returning False, I'm basically _disabling_ the feature, since it won't be shown in the dashboard.

After Some digging I found that, for some reason, rq and rq-scheduler organize scheduling and enqueuing of scheduled jobs in parallel ways, namely incompatible.

Please correct me if I'm wrong: Basically, jobs scheduled using Scheduler.enqueue_at will never be picked up by a worker --with-scheduler (*different* redis keys), and also the opposite is true. I wouldn't understand anyone having both running (and if they do, the worker will be worker only).

Recently, @cunla's django-rq-scheduler (>2022.12.1) transitioned to --with-scheduler.

My point is: showing that the Scheduler is active (worker --with-scheduler) and then scheduling jobs using rq-scheduler enqueue_at will cause stale jobs that never run and are never seen by django-rq.

What I propose is your original idea but only working if the rq-scheduler is not to be found, otherwise, the information would be misleading, to say the least.

Let me know

Reply:
@selwin As per my 

> What I propose is your original idea but only working if the rq-scheduler is not to be found, otherwise, the information would be misleading, to say the least.

@selwin as I mentioned, I prefer not to activate the feature for rq-scheduler, because it would be misleading.
I think the PR is ready to move forward. (At least to ship it as a half-feature)

However, I'm happy to make it according to your opinion.

Reply:
I agree, let's drop `rq-scheduler` altogether as it will only make things confusing.

Reply:
@selwin great! So the PR is good as it is. do you need to see a screenshot?

Reply:
Yes please, a screenshot would be good just to make sure that it works :)